examples/jsm/nodes/lighting/AnalyticLightNode.js

```diff
@@ -81,7 +85,9 @@ class AnalyticLightNode extends LightingNode {
 			const bias = reference( 'bias', 'float', shadow );
 			const normalBias = reference( 'normalBias', 'float', shadow );
 
-			let shadowCoord = uniform( shadow.matrix ).mul( positionWorld.add( normalWorld.mul( normalBias ) ) );
+			const position = object.material.shadowPositionNode || positionWorld;
+
+			let shadowCoord = uniform( shadow.matrix ).mul( position.add( normalWorld.mul( normalBias ) ) );
```

Comment:
Can we create an offset of the normals instead of changing the position? We could use it on other types of surfaces, this seems limiting to me since `normalWorld` continues with the reference from the previous position. 

An update I haven't made yet... I think we should test is that the leftover normals should follow the object's normalMap changes.